vpr/src/draw/draw_toggle_functions.cpp

```diff
@@ -442,8 +442,115 @@ void set_net_max_fanout_cbk(GtkSpinButton* self, ezgl::application* app) {
  */
 void set_net_alpha_value_cbk(GtkSpinButton* self, ezgl::application* app) {
     t_draw_state* draw_state = get_draw_state_vars();
-    draw_state->net_alpha = gtk_spin_button_get_value_as_int(self);
+    draw_state->net_alpha = (255 - gtk_spin_button_get_value_as_int(self))/255.0;
```

Comment:
I think the reason why net_alpha was originally described in a comment to be a ratio is because in the code the ratio is multiplied with draw_state->net_color[net_id].alpha which would hold the alpha value from the color that was assigned to the specific net. From a global search of (draw_state->net_color[net_id]) , I dont see anywhere where the transparency of a net's color was changed so this value should always just be 255 as long as it is not changed.

Reply:
However different colours may possibly have different transparencies, so I think its good to keep the code as it is.